src/pages/Search/EmptySearchView.tsx

```diff
@@ -144,11 +146,14 @@ function EmptySearchView({type, hasResults}: EmptySearchViewProps) {
             case CONST.SEARCH.DATA_TYPES.TRIP:
                 return {
                     headerMedia: LottieAnimations.TripsEmptyState,
-                    headerContentStyles: [StyleUtils.getWidthAndHeightStyle(375, 240), StyleUtils.getBackgroundColorStyle(theme.travelBG)],
+                    headerContentStyles: [
+                        isSmallScreen ? StyleUtils.getWidthAndHeightStyle(375, 240) : StyleUtils.getWidthAndHeightStyle('100%', '100%'),
```

Comment:
The way we're passing the styling doesn't look great. Why can't we apply the same styling across all platforms? Other Lottie animations work perfectly, so it feels like we're doing a bit of a patch-up by using `isSmallScreen` specific styles.

Reply:
@jayeshmangwani I try to use the same styling as other Lottie animations and update my PR. Thanks.

Reply:
> I try to use the same styling as other Lottie animations and update my PR. Thanks.

Sorry, I didn't catch that, are you planning to push your changes?

Reply:
I updated this part to have a min-width of 400, like other Lottie animations. And it seems to be working.

```tsx 

 headerContentStyles: [StyleUtils.getWidthAndHeightStyle('100%', '100%'), StyleUtils.getBackgroundColorStyle(theme.travelBG), {minWidth: 400}],
 
```

Reply:
I am testing these changes on all platforms now.